multiqc/utils/search_patterns.yaml

```
@@ -459,6 +459,10 @@ rna_seqc/coverage:
     fn_re: 'meanCoverageNorm_(high|medium|low)\.txt'
 rna_seqc/correlation:
     fn_re: 'corrMatrix(Pearson|Spearman)\.txt'
+rockhopper:
+    - fn: 'summary.txt'
+    - contents: 'Analyzing transcripts...'
+    - max_filesize: 500000
```

Making this a list gives each item `OR` logic, when I guess you want `AND`..

```suggestion
    fn: 'summary.txt'
    contents: 'Analyzing transcripts...'
    max_filesize: 500000
```

This is also super generic.. Are there no other strings in the file which are more specific? I'm a bit worried that this will match a lot of other tools. It's also good if you can limit the number of lines to search when looking for text strings (assuming that this isn't stdout that could be likely concatenated with other files).

---

I see, yes I would like AND logic.

As for the contents, it never actually states "Rockhopper", but perhaps the updated text may work better

---

Nice, this is much better - thanks 👍